building/export_plugin_from_eclipse.adoc

```diff
@@ -0,0 +1,37 @@
+= How to Export a plugin from Eclipse
+
+Exporting a plugin from inside eclipse is the 'quick and dirty' method to get an output binary for quick local tests. It does not generate an update site, and so this mechanism is only really suitable for local smoke-testing. 
```

Comment:
If anyone reads "quick and dirty" and thinks "this must be the way we build releases", they shouldn't be on this team. There's nothing quick about how we build releases. *ba-dump-cha*